### PR TITLE
CVS-59213 C++ example client for resnet

### PR DESCRIPTION
**Pull request by dkalinowski**

### CODE REVIEW

File: example_client/cpp/.bazelrc [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#
+# Copyright (c) 2020 Intel Corporation
```
Comment (atobiszei) [inline example_client/cpp/.bazelrc:2]: 2021

File: example_client/cpp/WORKSPACE [hunk 2]
```diff
@@ -0,0 +1,105 @@
+#
+# Copyright (c) 2020 Intel Corporation
```
Comment (atobiszei) [inline example_client/cpp/WORKSPACE:2]: 2021

File: example_client/cpp/.bazelrc (hunk 1 above)
Comment (dkalinowski) [reply example_client/cpp/.bazelrc]: done

File: example_client/cpp/WORKSPACE (hunk 2 above)
Comment (dkalinowski) [reply example_client/cpp/WORKSPACE]: done

**Review by dtrawins — APPROVED** [review]

**Review by atobiszei — APPROVED** [review]

**Pull request merged**